lib/lightning/jobs/job.ex

```diff
@@ -41,8 +41,7 @@ defmodule Lightning.Jobs.Job do
   @primary_key {:id, :binary_id, autogenerate: true}
   @foreign_key_type :binary_id
   schema "jobs" do
-    field :body, :string,
-      default: "// Get started by adding operations from your adaptor here"
```

Comment:
@stuartc just wanna flag this diff before I merge down - I've removed the default `body` setting for a job. It makes it easier for the client to distinguish placeholder nodes from real ones.

Does this seem alright?